QUESTIONS.md

```diff
@@ -1,5 +1,19 @@
 # Questions about Code, Node, Express, React
 
+- Can you talk about file IO and elaborate on it
+- Can you teach us more on React and what its functions are
+- What is the difference between Angular and React
+- How often is new code created vs just updating existing code in a typical work environment
+- What is a relational database?
```

Comment:
Short answer:
Relational works via tables where properties are RELATED by certain key fields

Nonrelational works via collections where "documents' (basically javascript obeects) with ALL properties are stored as one big thing.

Long answer: watch this incredible video: https://www.youtube.com/watch?v=ZS_kXvOeQ5Y